third_party/java/arcs/build_defs/internal/kotlin.bzl

```diff
@@ -266,6 +268,27 @@ def arcs_kt_particles(
             visibility = visibility,
         )
 
+def arcs_kt_plan(name, src, deps = [], out = None, visibility = None):
+    """Converts recipes in manifests into Kotlin Plans.
+
+    Args:
+      name: the name of the target to create
+      src: an Arcs manifest file
+      deps: list of dependencies (other manifests)
+      out: the name of the output artifact (a Kotlin file).
+      visibility: list of visibilities
+    """
+    outs = [out] if out != None else [output_name(name, ".kt")]
```

Comment:
output_name is supposed to clean up the file name of the Arcs manifest, but here you give it `name` not `src`. What's up?

Reply:
That's a good catch. It seems like `output_name` has been applied to `name` elsewhere, too, and I was following that pattern. I'll change it for all our rules.